boba-community/README.md

```diff
@@ -0,0 +1,80 @@
+# Running a replica node
+
+## Docker configuration
+
+Here are instructions if you want to run boba erigon version as the replica node for OP Mainnet or Testnet.
+
+### Get the data dir
+
+The first step is to download the initial data for `op-erigon`. Thanks for the contribution from [Test in Prod](https://www.testinprod.io).
+
+1. Download the correct data directory snapshot.
+
+* [OP Mainnet](https://op-erigon-backup.mainnet.testinprod.io)
+* [OP Goerli](https://op-erigon-backup.goerli.testinprod.io)
+
+2. Create the data directory in `op-erigon` and fill it.
+
+  ```bash
+  mkdir op-erigon
+  cd ./op-erigon
+  mkdir erigon
+  cd ./erigon
+  tar xvf ~/[DIR]/op-erigon-goerli.tar
+  ```
+
+3. Create a shared secret (JWT token)
+
+  ```bash
+  cd op-erigon
+  openssl rand -hex 32 > jwt.txt
```

Comment:
Here we use `jwt.txt` but below, we use `jwt-secret.txt`

Reply:
Updated